contracts/VToken.sol

```diff
@@ -881,18 +882,18 @@ contract VToken is
             /*
              * We get the current exchange rate and calculate the amount to be redeemed:
              *  redeemTokens = redeemAmountIn / exchangeRate
-             *  redeemAmount = redeemAmountIn
+             *  redeemAmount = exchangeRate * redeemTokens
              */
             redeemTokens = div_(redeemAmountIn, exchangeRate);
 
             uint256 _redeemAmount = mul_(redeemTokens, exchangeRate);
             if (_redeemAmount != 0 && _redeemAmount != redeemAmountIn) redeemTokens++; // round up
-            redeemAmount = redeemAmountIn;
+            redeemAmount = mul_ScalarTruncate(exchangeRate, redeemTokens);
```

Comment:
we can move this line and [this](https://github.com/VenusProtocol/isolated-pools/pull/247/files#diff-74967185a2c5eedfa0a1e0b396c10b82321afb533d1fca50ca5d896e81915e17R880) outside the if and else block as it is common.

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/247#discussion_r1229460347